src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 16px;
+  &__stars-text {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $main-text-color;
+  }
+}
+
+.stars {
+  display: flex;
+  &__star {
+    display: flex;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    margin-right: 4px;
+    &:nth-child(-n + 4) {
```

Comment:
Please, add css from the previous task to keep modifiers working

Reply:
hi, sorry I don't understand what you mean

Reply:
The stars block should have 2 classes: `stars stars--4` and the number of active stars should depend on it. The class `stars stars--1` should show 1 active star